.github/workflows/test_dbt_source_freshness.yaml

```diff
@@ -0,0 +1,28 @@
+name: test-dbt-source-freshness
```

Comment:
We had some discussion out of band about where/when would be the best time to run these checks. I'm starting with the simplest approach (scheduled once per day) but it should be easy to refactor this to e.g. allow sqoop to dispatch this workflow itself if we decide to move in that direction in the future.